app/src/components/Modals/add-activity-modal.tsx

```diff
@@ -181,7 +185,23 @@ const AddActivityModal: FC<AddUserModalProps> = ({
     });
   };
 
-  const formInputs = formStruct.formInputs;
+  const formInputs = formStruct?.formInputs[step][scope];
+
+  const [isEmissionFactorInputDisabled, setIsEmissionFactorInputDisabled] =
+    useState<boolean>(true);
+
+  const onEmissionFactorTypeChange = (e: any) => {
+    const emissionFactorType = e.target.value;
+    if (
+      emissionFactorType === "Local" ||
+      emissionFactorType === "Regional" ||
+      emissionFactorType === "National"
```

Comment:
This condition might need to be adjusted once we have data for national emissions factors (e.g. for the US we already do I believe).

Reply:
Great i'll leave a comment here